src/components/dato/PeriodeÅrMånedvelgere.tsx

```diff
@@ -119,7 +119,7 @@ const PeriodeÅrMånedvelgere: FC<Props> = ({
   return (
     <>
       <FeltGruppe>
-        <Label>{tekst}</Label>
+        <Label as="p">{tekst}</Label>
```

Comment:
Hvis denne `as` trengs for UU, burde ikke den väre required i designsystemet? 👁️

Reply:
Hvis du bruker `Label` i et form med en `label-for-input-element` så trenger du ikke `as`. Men hvis du bruker `Label` for å peke på en tekst-verdi så må den være en `p`.